app/constants/AppConstants.js

```diff
@@ -1,42 +1,48 @@
+import settings from '../../config/settings';
+
 export default {
-  API_URL: SETTINGS.API_URL,
+  API_URL: settings.API_URL,
   CUSTOMIZATIONS: {
     'varaamo.espoo.fi': 'ESPOO',
     'varaamotest-espoo.hel.ninja': 'ESPOO',
     'varaamo.vantaa.fi': 'VANTAA',
-    'varaamotest-vantaa.hel.ninja': 'VANTAA',
+    'varaamotest-vantaa.hel.ninja': 'VANTAA'
```

Comment:
Is your editor automatically removing these dangling commas? I propose that you double check that this happens intentionally. Personally I prefer having comma-dangling on since diffs are easier to read. 

Something seems to be off with Varaamo's eslint settings

Reply:
It's currently turned off in the `.eslintrc` file.